doc/contributor/howto-guide-generated-code-maintenance.md

```diff
@@ -81,19 +81,32 @@ Then send a PR with whatever changed.
 
 ## Bump all version numbers
 
+Manually bump the version of `google-cloud-gax-internal`, if necessary, in:
+
+- `Cargo.toml`
+- `src/gax-internal/Cargo.toml`
+
 Run:
 
 ```bash
+git fetch upstream
 git checkout -b chore-bump-version-numbers-circa-$(date +%Y-%m-%d)
 V=$(cat .sidekick-version.txt)
 go run github.com/googleapis/librarian/cmd/sidekick@${V} rust-bump-versions
-# The previous command fails when `gax-internal` has changed.
 git ls-files -z -- \
     '*.toml' ':!:**/testdata/**' ':!:**/generated/**' | \
     xargs -0 taplo fmt
-git commit -m"chore: bump version numbers circa $(date +%Y-%m-%d)" .
+# A crate's README depends on the crate's version. We need to regenerate them.
```

Comment:
Please open a bug with sidekick. This is not something we should need to remember (fine to merge this PR until that bug is fixed).